_episodes/03-matplotlib.md

```diff
@@ -13,15 +13,28 @@ keypoints:
 ---
 
 ## Visualizing data
-The mathematician Richard Hamming once said, "The purpose of computing is insight, not numbers," and
+The mathematician Richard Hamming once said, "The purpose of computing is insight, not numbers", and
 the best way to develop insight is often to visualize data.  Visualization deserves an entire
 lecture of its own, but we can explore a few features of Python's `matplotlib` library here.  While
 there is no official plotting library, `matplotlib` is the _de facto_ standard.  First, we will
-import the `pyplot` module from `matplotlib` and use two of its functions to create and display a
+import the `pyplot` module from `matplotlib`, and then we will use two of its functions to create and display a
```

Comment:
@ldko, could you please comment on this change?

Reply:
I don't think we necessarily need this change, as "First" in the context of all the visualizations made in the episode would be first making the heatmap (needing to import `pyplot` to do that) and second being the mean plot that follows.